index.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style2.css">
+  <title>Document</title>
+</head>
+<body>
+    <hi>Gallery</hi>
+    <section class="imgsection"> 
+      <div class="imgwrapper">
+        <div class=imgcontainer>
+          <a class="project" href="#">
+            <figure class="figimg">
+              <img src="./imgs/leaves/george-vogiatzis-3YrjdBBckzM-unsplash.jpg" id="elixirlogo" alt="heart logo" class="item"></img>
+              <figcaption>
+                 <div>
```

Comment:
nothing big but the `divs` around the `h3` and `p` tag seem a bit unnecessary and I don't see any code specifically for them. you don't have to remove now but just something to think about in the future.